core/components/footer/footer.tsx

```diff
@@ -1,39 +1,56 @@
 import { removeEdgesAndNodes } from '@bigcommerce/catalyst-client';
-import {
-  SiFacebook,
-  SiInstagram,
-  SiLinkedin,
-  SiPinterest,
-  SiX,
-  SiYoutube,
-} from '@icons-pack/react-simple-icons';
-import { JSX } from 'react';
 
+import { Footer as FooterSection } from '@/vibes/soul/sections/footer';
+import {
+  Amex,
+  ApplePay,
+  Bitcoin,
+  GooglePay,
+  Mastercard,
+  Paypal,
+  Visa,
+} from '@/vibes/soul/sections/footer/payment-icons';
+import { Facebook, Instagram, X, Youtube } from '@/vibes/soul/sections/footer/social-icons';
 import { LayoutQuery } from '~/app/[locale]/(default)/query';
 import { getSessionCustomerAccessToken } from '~/auth';
 import { client } from '~/client';
 import { readFragment } from '~/client/graphql';
 import { revalidate } from '~/client/revalidate-target';
-import { Footer as ComponentsFooter } from '~/components/ui/footer';
-import { logoTransformer } from '~/data-transformers/logo-transformer';
 
 import { FooterFragment } from './fragment';
-import { AmazonIcon } from './payment-icons/amazon';
-import { AmericanExpressIcon } from './payment-icons/american-express';
-import { ApplePayIcon } from './payment-icons/apple-pay';
-import { MastercardIcon } from './payment-icons/mastercard';
-import { PayPalIcon } from './payment-icons/paypal';
-import { VisaIcon } from './payment-icons/visa';
 
-const socialIcons: Record<string, { icon: JSX.Element }> = {
-  Facebook: { icon: <SiFacebook title="Facebook" /> },
-  Twitter: { icon: <SiX title="Twitter" /> },
-  X: { icon: <SiX title="X" /> },
-  Pinterest: { icon: <SiPinterest title="Pinterest" /> },
-  Instagram: { icon: <SiInstagram title="Instagram" /> },
-  LinkedIn: { icon: <SiLinkedin title="LinkedIn" /> },
-  YouTube: { icon: <SiYoutube title="YouTube" /> },
-};
+const copyright = `© ${new Date().getFullYear().toString()} SOUL - Powered by Monogram`;
+
+const logo = 'SOUL';
+
+const paymentIconsArray: React.ReactNode[] = [
+  <Visa key="Visa" />,
+  <Amex key="Amex" />,
+  <Mastercard key="Mastercard" />,
+  <Paypal key="Paypal" />,
+  <GooglePay key="GooglePay" />,
+  <ApplePay key="ApplePay" />,
+  <Bitcoin key="Bitcoin" />,
```

Comment:
These icon seem to be different from the original ones. In particular, we've removed the Amazon and added the Bitcoin and Google Pay icons. Are these payment methods actually supported during checkout? We probably want to keep what was already there before in `main`.

Or @bookernath, maybe you can speak to what we should and shouldn't have here.